graphql/playground/apollo_sandbox_playground.go

```diff
@@ -0,0 +1,57 @@
+package playground
+
+import (
+	"html/template"
+	"net/http"
+)
+
+var apolloSandboxPage = template.Must(template.New("ApolloSandbox").Parse(`<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>{{.title}}</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link rel="icon" href="https://embeddable-sandbox.cdn.apollographql.com/_latest/public/assets/favicon-dark.png">
+	<style>
+	body {
+		margin: 0;
+		overflow: hidden;
+	}
+</style>
+</head>
+
+<body>
+  <div style="width: 100vw; height: 100vh;" id='embedded-sandbox'></div>
+  <!-- NOTE: New version available at https://embeddable-sandbox.cdn.apollographql.com/ -->
+  <script rel="preload" as="script" crossorigin="anonymous" integrity="{{.mainSRI}}" type="text/javascript" src="https://embeddable-sandbox.cdn.apollographql.com/58165cf7452dbad480c7cb85e7acba085b3bac1d/embeddable-sandbox.umd.production.min.js"></script>
+  <script>
+{{- if .endpointIsAbsolute}}
+	const url = {{.endpoint}};
+{{- else}}
+	const url = location.protocol + '//' + location.host + {{.endpoint}};
+{{- end}}
+  new window.EmbeddedSandbox({
+    target: '#embedded-sandbox',
+    initialEndpoint: url,
+		persistExplorerState: true,
+  });
+  </script>
+</body>
+
+</html>`))
+
+// ApolloSandboxHandler responsible for setting up the altair playground
+func ApolloSandboxHandler(title, endpoint string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		err := apolloSandboxPage.Execute(w, map[string]interface{}{
+			"title":              title,
+			"endpoint":           endpoint,
+			"endpointIsAbsolute": endpointHasScheme(endpoint),
+			"mainSRI":            "sha256-/E4VNgAWFmbNLyXACSYoqsDAj68jC1sCMSQ0cDjf4YM=",
```

Comment:
Yes, I followed the MDN documentation to get the hash value locally because I figured I needed to specify the RSI from another Handler implementation.
https://github.com/99designs/gqlgen/blob/20ec0ee734c98d371dd1cf2c85314098b2650dd3/graphql/playground/altair_playground.go#L41

I dit it.

```sh
CDN_FILE=https://embeddable-sandbox.cdn.apollographql.com/58165cf7452dbad480c7cb85e7acba085b3bac1d/embeddable-sandbox.umd.production.min.js
curl -s $CDN_FILE | openssl dgst -sha256 -binary | openssl base64 -A; echo
```

If subresource integrity is not required, it may be more convenient to always specify the most current resource.

Reply:
> Is this something that needs to be updated along with the JS CDN link?

Yes, because the version is fixed by the commit hash. The integrity hash must be recomputed each time the version is bumped.